website/docs/d/kms_secrets.html.markdown

```diff
@@ -17,11 +17,14 @@ Decrypt multiple secrets from data encrypted with the AWS KMS service.
 If you do not already have a `CiphertextBlob` from encrypting a KMS secret, you can use the below commands to obtain one using the [AWS CLI kms encrypt](https://docs.aws.amazon.com/cli/latest/reference/kms/encrypt.html) command. This requires you to have your AWS CLI setup correctly and replace the `--key-id` with your own. Alternatively you can use `--plaintext 'master-password'` (CLIv1) or `--plaintext fileb://<(echo -n 'master-password')` (CLIv2) instead of reading from a file.
 
 -> If you have a newline character at the end of your file, it will be decrypted with this newline character intact. For most use cases this is undesirable and leads to incorrect passwords or invalid values, as well as possible changes in the plan. Be sure to use `echo -n` if necessary.
-
+-> If you are using asymmetric keys ensure you are using the right encryption algorithm when you encrypt
+and decrypt else you will get IncorrectKeyException during the decrypt phase.
```

Comment:
We can combine these lines into 1

```suggestion
-> If you are using asymmetric keys ensure you are using the right encryption algorithm when you encrypt and decrypt else you will get IncorrectKeyException during the decrypt phase.

Reply:
In my VS Code, it looks as a single line only!!!